app/code/Magento/ProductAlert/Controller/Unsubscribe/Stock.php

```diff
@@ -21,16 +22,33 @@ class Stock extends UnsubscribeController
     protected $productRepository;
 
     /**
-     * @param \Magento\Framework\App\Action\Context $context
-     * @param \Magento\Customer\Model\Session $customerSession
-     * @param \Magento\Catalog\Api\ProductRepositoryInterface $productRepository
+     * @var Magento\Store\Model\StoreManagerInterface
```

Comment:
It looks like you have typo `Magento\Store\Model\StoreManagerInterface` instead of `\Magento\Store\Model\StoreManagerInterface`.

Could you also simplify to short `StoreManagerInterface`?